<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1880</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="studentsTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1552</width>
      <height>880</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="findButton">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>40</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Chose file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>90</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load from file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>140</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>190</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add person</string>
    </property>
   </widget>
   <widget class="QPushButton" name="correctButton">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>240</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Correct person</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>290</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete person</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>340</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort and Find</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1880</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
